### PR TITLE
[Review needed] Create 03-2021-CW21LT.md

### LINKED ISSUE #1749: Talk proposal: Lightning talk at CW21 on the Activism Chapter of the Ethics Book - deadline 7 March
### Summary

Issue to discuss Lightning Talk proposal for the "Activism" chapter of the Ethics Book for the Software Sustainability Institute's Collaborations Workshop 2021. 

Deadline 7 March 

Guidelines and more information here: https://www.software.ac.uk/news/call-lightning-talks-collaborations-workshop-2021 

### What needs to be done?

- [ ] Draft title
- [ ] Draft abstract
- [ ] Feedback on title & abstract
- [ ] Submit! **deadline 7 March 2021**
- [x] *make sure the chapter is live by then....*
- [ ] Draft slide & Lightning Talk
- [ ] Feedback on slide & Lightning Talk 

### Who can help?

* Contributors to the Ethics book or ethics discussions particularly welcome: @LauraCarter, @Arielle-Bennett, @irenekp, @Ismael-KG   

---

### Updates

*Updates when we have them.*

### PR DESCRIPTION
**Pull request by Arielle-Bennett**
### Summary

Fixes #1749

Adding Lightning Talk abstract for Activism to be submitted to CW21

### List of changes proposed in this PR (pull-request)

* Adding Lightning Talk abstract for CW21 to the conferences folder 

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [X] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

File: conferences/abstracts/03-2021-CW21LT.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+If you want to use a lightning talk slot to introduce yourself and your work, then the title can simply be your full name!*
+
+Introducing the Activism Chapter of the _Turing Way Guide to Ethical Research_. 
+
+## Lightning talk abstract
+*Just 2-3 sentences to add context to your lightning talk title in Question 4. This will not be published anywhere, but will help us to review your submission.*
+
+There are many different scenarios where scientists and researchers would engage in activism to achieve change, such as increasing ethical standards of research, improving working conditions, or tackling systemic inequalities. 
+Scientists may engage in activism across topics related to their work, institutions, fields, cross-disciplinary issues such as open research, or on topics unrelated to their career.
+However, there are limited guides accessible to those who would like to achieve change but are unsure where to begin, which the latest chapter of the _Turing Way Guide to Ethical Research_ seeks to offer.
```
Comment (LauraCarter) [inline conferences/abstracts/03-2021-CW21LT.md:14]:
```suggestion
However, there are limited guides accessible to those who would like to achieve change but are unsure where to begin
The latest chapter of the _Turing Way Guide to Ethical Research_ aims to fill that gap.
The Activism chapter aims to help scientists engage in activism across topics related to their work, institutions, fields, cross-disciplinary issues such as open research, or even on topics unrelated to their career.
```

File: conferences/abstracts/03-2021-CW21LT.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+*For example, the name of the project, initiative or topic you want to present. 
+If you want to use a lightning talk slot to introduce yourself and your work, then the title can simply be your full name!*
+
+Introducing the Activism Chapter of the _Turing Way Guide to Ethical Research_. 
+
+## Lightning talk abstract
+*Just 2-3 sentences to add context to your lightning talk title in Question 4. This will not be published anywhere, but will help us to review your submission.*
+
+There are many different scenarios where scientists and researchers would engage in activism to achieve change, such as increasing ethical standards of research, improving working conditions, or tackling systemic inequalities. 
+Scientists may engage in activism across topics related to their work, institutions, fields, cross-disciplinary issues such as open research, or on topics unrelated to their career.
```
Comment (LauraCarter) [inline conferences/abstracts/03-2021-CW21LT.md:13]:
```suggestion
```

File: conferences/abstracts/03-2021-CW21LT.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+However, there are limited guides accessible to those who would like to achieve change but are unsure where to begin, which the latest chapter of the _Turing Way Guide to Ethical Research_ seeks to offer.
+
+This chapter welcomes additional contributions and case studies as it continues to expand to include examples of data science and tech activism from across the globe.
```
Comment (LauraCarter) [inline conferences/abstracts/03-2021-CW21LT.md:16]:
```suggestion
This chapter welcomes additional contributions and case studies as it continues to expand to support activism in data science and tech across the globe. 
```

**Review by LauraCarter — COMMENTED** [review]
Looks good! I've made some suggested edits to the abstract, feel free to include/ignore as you wish!

File: conferences/abstracts/2021-03-CW21LT.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Submitting a Lightning Talk Abstract on the Activism Chapter in the Ethics Book to SSIs Collaborations Workshop 2021
```
Comment (malvikasharan) [inline conferences/abstracts/2021-03-CW21LT.md:1]: Add the following details on the top 

```
# Collaboration Workshop 2021

* Website: https://www.software.ac.uk/cw21
* Location: Online
* Date: 30 March - 01 April 2021

## Authors
Arielle Bennette

## Session collaborators
Laura Carter
```

File: conferences/abstracts/2021-03-CW21LT.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+## Lightning talk abstract
+*Just 2-3 sentences to add context to your lightning talk title in Question 4. This will not be published anywhere, but will help us to review your submission.*
+
+There are many different scenarios where scientists and researchers would engage in activism to achieve change, such as increasing ethical standards of research, improving working conditions, or tackling systemic inequalities. 
+Scientists may engage in activism across topics related to their work, institutions, fields, cross-disciplinary issues such as open research, or on topics unrelated to their career.
+However, there are limited guides accessible to those who would like to achieve change but are unsure where to begin, which the latest chapter of the _Turing Way Guide to Ethical Research_ seeks to offer.
+
+This chapter welcomes additional contributions and case studies as it continues to expand to include examples of data science and tech activism from across the globe. 
+You can find the Activism Chapter here: URL TO FOLLOW
```
Comment (malvikasharan) [inline conferences/abstracts/2021-03-CW21LT.md:17]:
```suggestion
You can find the Activism Chapter here: https://the-turing-way.netlify.app/ethical-research/activism.html
```

File: conferences/abstracts/2021-03-CW21LT.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+If you want to use a lightning talk slot to introduce yourself and your work, then the title can simply be your full name!*
+
+Introducing the Activism Chapter of the _Turing Way Guide to Ethical Research_. 
+
+## Lightning talk abstract
+*Just 2-3 sentences to add context to your lightning talk title in Question 4. This will not be published anywhere, but will help us to review your submission.*
+
+There are many different scenarios where scientists and researchers would engage in activism to achieve change, such as increasing ethical standards of research, improving working conditions, or tackling systemic inequalities. 
+Scientists may engage in activism across topics related to their work, institutions, fields, cross-disciplinary issues such as open research, or on topics unrelated to their career.
+However, there are limited guides accessible to those who would like to achieve change but are unsure where to begin, which the latest chapter of the _Turing Way Guide to Ethical Research_ seeks to offer.
```
Comment (malvikasharan) [inline conferences/abstracts/2021-03-CW21LT.md:14]:
```suggestion
There are many circumstances where researchers engage in activism to achieve scientific and structural change, such as maintaining ethical standards in research, advocating for human rights, improving working conditions, or tackling systemic inequalities. 
In fact, we all can engage in activism across topics related to our work, institutions, research practices, cross-disciplinary issues such as open research, workplace policy or on topics unrelated to their career.
However, there are limited accessible resources that provide guidance on how to lead or participate in civic activism, leaving people unsure of where to begin.
In this lightning talk, I would introduce the latest chapter "Activism for Researchers" in the _Turing Way Guide for Ethical Research_ and invite contributions in the form of examples, case studies and feedback.
```

**Review by malvikasharan — APPROVED** [review]
@Arielle-Bennett This looks great! I have added some suggestions, accept only what works for you. Mainly, I have reordered your second sentence to be the opening line - maybe not entirely necessary.

**Pull request merged**